Misc/NEWS.d/next/Library/2024-07-14-09-56-14.gh-issue-109523.lj6dfp.rst

```diff
@@ -0,0 +1 @@
+stdin.read() no longer throws a TypeError if no input is provided when stdin is set to non-blocking.
```

Comment:
```suggestion
``sys.stdin.read()`` no longer throws a :exc:`TypeError` if no input is provided
when :data:`sys.stdin` is set to non-blocking.
```

I'm wondering whether the change should only mention `sys.stdin` or not. Here, it's for *any* `TextIOWrapper`.